modules/understanding-telemetry-and-insights-operator-data-flow.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/about-remote-health-monitoring.adoc
+
+[id="understanding-telemetry-and-insights-operator-data-flow_{context}"]
+= Understanding Telemetry and Insights Operator data flow
+
+The Insights Operator gathers data from the Kubernetes API and the Prometheus API into an archive. The archive is uploaded to link:https://cloud.redhat.com[cloud.redhat.com] every four minutes for processing. The Insights Operator also downloads the latest Insights analysis from link:https://cloud.redhat.com/openshift/[cloud.redhat.com]. This is used to populate the *Insights status* pop-up that is included in the *Overview* page in the {product-title} web console.
```

Comment:
Maybe worth mentioning "gathering *selected* data", to reinforce that we don't pull everything. That's true both for IO and telemetry

Reply:
Great suggestion, thank you. Resolved.